cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/NvaProjectBuilder.java

```diff
@@ -150,9 +151,14 @@ public NvaProject build() {
                    .withApprovals(extractApprovals(cristinProject.getApprovals()))
                    .withExemptFromPublicDisclosure(cristinProject.getExemptFromPublicDisclosure())
                    .withCreator(extractCreator(cristinProject.getCreator()))
+                   .withWebPage(extractWebPage(cristinProject.getExternalUrl()))
                    .build();
     }
 
+    private URI extractWebPage(String externalUrl) {
```

Comment:
Right the optional of getter does make sense

Reply:
no other field in this model class is using optional. if we want to start using it i suggest we do it for all optional fields in a separate PR. we should also discuss one day using field = null or field omitted if there is no value there.